CMakeLists.txt

```diff
@@ -10,11 +10,99 @@ cmake_minimum_required(VERSION 3.16)
 if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE "Debug")
 endif()
+
```

Comment:
Please add another line with `SPDX-FileContributor: Be <be.0@gmx.com>` at the top :-)